packages/open-payments/src/client/requests.test.ts

```diff
@@ -175,7 +175,17 @@ describe('requests', (): void => {
         id: 'id'
       }
 
-      nock(baseUrl).post('/grant', body).reply(status, body)
+      const scope = nock(baseUrl)
+        .matchHeader('Signature', /sig1=:([a-zA-Z0-9+/]){86}==:/)
+        .matchHeader(
+          'Signature-Input',
+          `sig1=("@method" "@target-uri" "content-digest" "content-length" "content-type");created=${Math.floor(
+            Date.now() / 1000
```

Comment:
This test also needs
https://github.com/interledger/rafiki/blob/632bdfb15a920062637dc8a273034e58b382205b/packages/open-payments/src/client/requests.test.ts#L45-L58